Week2/homework/trigger.sql

```diff
@@ -0,0 +1,18 @@
+
+delimiter $$
+CREATE TRIGGER alertOnCountryCount
```

Comment:
Yes it's good only in this case I would have done the trigger before the insert and not after.